src/App/App.js

```diff
@@ -20,10 +20,59 @@ class App extends Component {
     };
   }
 
+  searchMovies = async (query) => {
+    query = query.toLowerCase()
+    const searchQueries = query.split(' ')
+    const allMovies = await this.showSearchResultsPage()
+    const checkedMovies = []
+    await allMovies.forEach(async (movie) => {
+      let fullMovie
+      try {
+        fullMovie = await api.getAMovie(movie.id)
+        fullMovie.year = await fullMovie ? fullMovie.release_date.substring(0, 4) : null;
+        if (this.checkAllQueriesAgainstMovie(searchQueries, fullMovie)) {
+          checkedMovies.push(fullMovie)
+          this.setState({movies: checkedMovies})
+        }
+      } catch (error) {
+        console.log('You\'ve got the following error - you gotta program something to do with it bubbua:', error.message)
+      }
+      this.setState({movies: checkedMovies})
+    })
+  }
+
+  checkAllQueriesAgainstMovie(searchQueries, movie) {
+    debugger
+    movie = Object.values(movie)
```

Comment:
Are you finished with the search stuff? Can we clean up debugger and what psuedo?